src/index.html

```diff
@@ -1,13 +1,413 @@
+            Mazel and VA Tsukkerman in his "Analysis
+            of musical works." <br> Pointillism, which
+            originated in the music of the early
+            twentieth <br> century, microforms, found
+            a distant historical. <br> <br>
+          </p>
+          <p class="description__text-quote">
+            The market leader in real estate since 2003.
+            Awards Real Estate <br> Company, 2010, 2011, 2013.
+            More than 200 satisfied customers <br> premium
+            segment. We focus only on new buildings and
+            high-level <br> doskanalno know this area better
+            than other specialists. <br> Working directly
+            with developers - so have more useful
+            information <br> about the profitable deals. <br> <br>
+          </p>
+          <p class="description__text-guarantee">
+            We will never be late! <br>
+            Guaranteed secure transaction - 100%
+          </p>
+        </div>
+      </div>
+    </section>
+
+    <section class="gallery">
+      <h2 class="gallery__title">
+        Gallery
+      </h2>
+      <img
+        src="images/crown.svg"
+        alt="crown image"
+        class="gallery__crown-img"
+      >
+      <div class="gallery__wrap">
+        <div class="gallery__marina">
+          <img
+            src="images/photos/1.jpg"
+            alt="house` photo"
+            width="673px"
+            height="256px"
+            class="gallery__img-1"
+          >
+          <img
+            src="images/photos/2.jpg"
+            alt="house` photo"
+            width="327px"
+            height="256px"
+            class="gallery__img-2"
+          >
+          <p class="gallery__text">
+            <span class="gallery__span">Marina Palms</span> / North Miami Beach, FL 33162
+          </p>
+        </div>
+        <div class="gallery__edition">
+          <img
+            src="images/photos/3.jpg"
+            alt="house` photo"
+            width="500px"
+            height="256px"
+            class="gallery__img-1"
+        >
+        <img
+          src="images/photos/4.jpg"
+          alt="house` photo"
+          width="500px"
+          height="256px"
+          class="gallery__img-2"
+        >
+        <p class="gallery__text">
+          <span class="gallery__span">
+            Edition Residens
+          </span> / Miami Beach, FL 33139
+        </p>
+        </div>
+        <div class="gallery__faena">
+          <img
+            src="images/photos/5.jpg"
+            alt="house` photo"
+            width="327px"
+            height="256px"
+            class="gallery__img-1"
+        >
+        <img
+          src="images/photos/6.jpg"
+          alt="house` photo"
+          width="673px"
+          height="256px"
+          class="gallery__img-2"
+        >
+        <p class="gallery__text">
+          <span class="gallery__span">
+            Faena House
+          </span> / Miami Beach, FL 33140
```

Comment:
```suggestion
          </span> / Miami Beach, FL 33140
```
https://www.w3schools.com/tags/tag_address.asp

Reply:
fixed